diff --git a/src/declarative/particles/qsgtargetaffector.cpp b/src/declarative/particles/qsgtargetaffector.cpp
index c5b5b07..01ec016 100644
--- a/src/declarative/particles/qsgtargetaffector.cpp
+++ b/src/declarative/particles/qsgtargetaffector.cpp
@@ -76,7 +76,7 @@ bool QSGTargetAffector::affectParticle(QSGParticleData *d, qreal dt)
     qreal vY = (target.y() - d->y) / tt;
 
     qreal w = 1 - (t / tt) + 0.05;
-    w = qMin(w, 1.0);
+    w = qMin(w, qreal(1.0));
     qreal wvX = vX * w + d->sx * (1 - w);
     qreal wvY = vY * w + d->sy * (1 - w);
     //Screws with the acceleration so that the given start pos with the chosen weighted velocity will still end at the target coordinates
