diff --git a/src/declarative/declarative.pro b/src/declarative/declarative.pro
index aa638e4..a816686 100644
--- a/src/declarative/declarative.pro
+++ b/src/declarative/declarative.pro
@@ -52,4 +52,8 @@ linux-g++-maemo:DEFINES += QDECLARATIVEVIEW_NOBACKGROUND
 DEFINES += QT_NO_OPENTYPE
 INCLUDEPATH += $$QT_SOURCE_TREE/src/3rdparty/harfbuzz/src
 
+# We install V8 headers into the system, under qt5/QtDeclarative/v8.
+# This requires a modification to v8.h include path
+INCLUDEPATH += ../3rdparty/
+
 
diff --git a/src/declarative/qml/v8/qv8_p.h b/src/declarative/qml/v8/qv8_p.h
index 6aeb6f6..6acf47d 100644
--- a/src/declarative/qml/v8/qv8_p.h
+++ b/src/declarative/qml/v8/qv8_p.h
@@ -1 +1 @@
-#include "../../../3rdparty/v8/include/v8.h"
+#include <v8/include/v8.h>
diff --git a/src/imports/gestures/gestures.pro b/src/imports/gestures/gestures.pro
index 5c009dd..a3e23b4 100644
--- a/src/imports/gestures/gestures.pro
+++ b/src/imports/gestures/gestures.pro
@@ -24,3 +24,6 @@ symbian:{
 }
 
 INSTALLS += target qmldir
+
+# V8 headers
+INCLUDEPATH += ../../../src/3rdparty
diff --git a/src/imports/particles/particles.pro b/src/imports/particles/particles.pro
index c73d068..1734b68 100644
--- a/src/imports/particles/particles.pro
+++ b/src/imports/particles/particles.pro
@@ -31,3 +31,6 @@ symbian:{
 }
 
 INSTALLS += target qmldir
+
+# V8 headers
+INCLUDEPATH += ../../../src/3rdparty
diff --git a/src/imports/qtquick1/qtquick1.pro b/src/imports/qtquick1/qtquick1.pro
index 1c7e324..6370066 100644
--- a/src/imports/qtquick1/qtquick1.pro
+++ b/src/imports/qtquick1/qtquick1.pro
@@ -28,3 +28,6 @@ symbian:{
 }
 
 INSTALLS += target qmldir
+
+# V8 headers
+INCLUDEPATH += ../../../src/3rdparty
diff --git a/src/imports/testlib/testlib.pro b/src/imports/testlib/testlib.pro
index 9980a75..bf0e3e9 100644
--- a/src/imports/testlib/testlib.pro
+++ b/src/imports/testlib/testlib.pro
@@ -44,3 +44,6 @@ copy2build.output = $$QT.declarative.imports/$$TARGETPATH/${QMAKE_FILE_BASE}${QM
 
 
 INSTALLS += target otherImportFiles
+
+# V8 headers
+INCLUDEPATH += ../../../src/3rdparty
diff --git a/src/plugins/qmltooling/qmldbg_inspector/qmldbg_inspector.pro b/src/plugins/qmltooling/qmldbg_inspector/qmldbg_inspector.pro
index 7f4d54e..65351b0 100644
--- a/src/plugins/qmltooling/qmldbg_inspector/qmldbg_inspector.pro
+++ b/src/plugins/qmltooling/qmldbg_inspector/qmldbg_inspector.pro
@@ -61,3 +61,6 @@ target.path += $$[QT_INSTALL_PLUGINS]/qmltooling
 INSTALLS += target
 
 symbian:TARGET.UID3=0x20031E90
+
+# V8 headers
+INCLUDEPATH += ../../../../src/3rdparty
diff --git a/src/plugins/qmltooling/qmldbg_tcp/qmldbg_tcp.pro b/src/plugins/qmltooling/qmldbg_tcp/qmldbg_tcp.pro
index c5f8f90..8c854c1 100644
--- a/src/plugins/qmltooling/qmldbg_tcp/qmldbg_tcp.pro
+++ b/src/plugins/qmltooling/qmldbg_tcp/qmldbg_tcp.pro
@@ -18,3 +18,6 @@ target.path += $$[QT_INSTALL_PLUGINS]/qmltooling
 INSTALLS += target
 
 symbian:TARGET.UID3=0x20031E90
+
+# V8 headers
+INCLUDEPATH += ../../../../src/3rdparty
diff --git a/src/qtquick1/qtquick1.pro b/src/qtquick1/qtquick1.pro
index e3dd029..f0ec3eb 100644
--- a/src/qtquick1/qtquick1.pro
+++ b/src/qtquick1/qtquick1.pro
@@ -40,3 +40,7 @@ SOURCES += qtquick1.cpp
 DEFINES += QT_NO_OPENTYPE
 INCLUDEPATH += $$QT_SOURCE_TREE/src/3rdparty/harfbuzz/src
 
+# V8 header path changed for system install.
+# Add top-level dir for "v8/include/"
+INCLUDEPATH += ../../src/3rdparty/
+
diff --git a/tools/qmlplugindump/qmlplugindump.pro b/tools/qmlplugindump/qmlplugindump.pro
index 0ac3a70..a421567 100644
--- a/tools/qmlplugindump/qmlplugindump.pro
+++ b/tools/qmlplugindump/qmlplugindump.pro
@@ -35,3 +35,6 @@ CONFIG(debug, debug|release) {
 
 target.path = $$[QT_INSTALL_BINS]
 INSTALLS += target
+
+# V8 headers
+INCLUDEPATH += ../../src/3rdparty
diff --git a/tools/qmlscene/qmlscene.pro b/tools/qmlscene/qmlscene.pro
index 8e252e5..0874669 100644
--- a/tools/qmlscene/qmlscene.pro
+++ b/tools/qmlscene/qmlscene.pro
@@ -18,3 +18,6 @@ symbian {
 }
 
 DEFINES += QML_RUNTIME_TESTING
+
+# V8 headers
+INCLUDEPATH += ../../src/3rdparty
diff --git a/tools/qmltestrunner/qmltestrunner.pro b/tools/qmltestrunner/qmltestrunner.pro
index c431aaf..8177992 100644
--- a/tools/qmltestrunner/qmltestrunner.pro
+++ b/tools/qmltestrunner/qmltestrunner.pro
@@ -10,3 +10,6 @@ DEFINES += QUICK_TEST_SOURCE_DIR=\"\\\"$$OUT_PWD\\\"\"
 
 target.path = $$[QT_INSTALL_BINS]
 INSTALLS += target
+
+# V8 headers
+INCLUDEPATH += ../../src/3rdparty
diff --git a/tools/qmlviewer/qmlviewer.pro b/tools/qmlviewer/qmlviewer.pro
index 16579bb..a5ab713 100644
--- a/tools/qmlviewer/qmlviewer.pro
+++ b/tools/qmlviewer/qmlviewer.pro
@@ -11,6 +11,9 @@ INCLUDEPATH += ../../include/QtDeclarative
 INCLUDEPATH += ../../src/declarative/util
 INCLUDEPATH += ../../src/declarative/graphicsitems
 
+# V8 headers
+INCLUDEPATH += ../../src/3rdparty
+
 DEFINES += QT_NO_CAST_FROM_ASCII QT_NO_CAST_TO_ASCII
 
 target.path = $$[QT_INSTALL_BINS]
