diff --git a/src/declarative/declarative.pro b/src/declarative/declarative.pro
index a4aab38..5ffcece 100644
--- a/src/declarative/declarative.pro
+++ b/src/declarative/declarative.pro
@@ -55,5 +55,5 @@ INCLUDEPATH += $$QT_SOURCE_TREE/src/3rdparty/harfbuzz/src
 macx:CONFIG(debug, debug|release) {
     QMAKE_LIBS_PRIVATE += -L../v8/ -lv8_debug
 } else {
-    QMAKE_LIBS_PRIVATE += -L../v8/ -lv8
+    QMAKE_LIBS_PRIVATE += -L../v8/ -lv8 -lpthread
 }
