diff --git a/src/declarative/particles/qquickimageparticle.cpp b/src/declarative/particles/qquickimageparticle.cpp
index 49bdcb2..f5a5f31 100644
--- a/src/declarative/particles/qquickimageparticle.cpp
+++ b/src/declarative/particles/qquickimageparticle.cpp
@@ -193,7 +193,7 @@ static const char vertexShaderCode[] =
     "          + 0.5 * vVec.zw * pow(t * vData.y, 2.);\n"
     "    gl_PointSize = currentSize;\n"
     "#endif\n"
-    "    gl_Position = qt_Matrix * vec4(pos.x, pos.y, 0, 1);\n"
+    "    gl_Position = qt_Matrix * vec4(pos.x, pos.y, 0., 1.);\n"
     "\n"
     "#ifdef COLOR\n"
     "    fColor = vColor * fade;\n"
