diff --git a/src/phonon/phonon.pro b/src/phonon/phonon.pro
index db7605f..f45a4ff 100644
--- a/src/phonon/phonon.pro
+++ b/src/phonon/phonon.pro
@@ -14,7 +14,10 @@ DEFINES += MAKE_PHONON_LIB
 
 PHONON_DIR = ../3rdparty/phonon/phonon
 
-unix|win32-g++*:QMAKE_PKGCONFIG_REQUIRES = QtCore QtGui QtNetwork
+unix|win32-g++* {
+    QMAKE_PKGCONFIG_REQUIRES = QtCore QtGui QtNetwork
+    CONFIG += create_prl create_pc
+}
 
 # Input
 HEADERS += $$PHONON_DIR/abstractaudiooutput.h \
