diff --git a/src/location/location.pro b/src/location/location.pro
index 05c4eb3..bd9230c 100644
--- a/src/location/location.pro
+++ b/src/location/location.pro
@@ -12,6 +12,11 @@ DEFINES += QT_BUILD_LOCATION_LIB QT_MAKEDLL
 
 INCLUDEPATH += $$QT.location.private_includes/QtLocation
 
+unix|win32-g++* {
+    # QMAKE_PKGCONFIG_REQUIRES += ???
+    CONFIG += create_prl create_pc
+}
+
 load(qt_module_config)
 
 HEADERS += qtlocationversion.h
