diff --git a/src/imports/location/qdeclarativegeomapmousearea_p.h b/src/imports/location/qdeclarativegeomapmousearea_p.h
index 92174ab..3a2d775 100644
--- a/src/imports/location/qdeclarativegeomapmousearea_p.h
+++ b/src/imports/location/qdeclarativegeomapmousearea_p.h
@@ -46,7 +46,7 @@
 #include "qdeclarativegraphicsgeomap_p.h"
 #include "qdeclarativegeomapmouseevent_p.h"
 
-#include <QtDeclarative/qdeclarativeitem.h>
+#include <QtQuick1/qdeclarativeitem.h>
 
 QT_BEGIN_NAMESPACE
 
diff --git a/src/imports/location/qdeclarativegeomapobject_p.h b/src/imports/location/qdeclarativegeomapobject_p.h
index 26904df..a2068b2 100644
--- a/src/imports/location/qdeclarativegeomapobject_p.h
+++ b/src/imports/location/qdeclarativegeomapobject_p.h
@@ -47,7 +47,7 @@
 #include "qgeomapgroupobject.h"
 #include "QModelIndex"
 
-#include <QtDeclarative/qdeclarativeitem.h>
+#include <QtQuick1/qdeclarativeitem.h>
 class QAbstractItemModel;
 
 QT_BEGIN_NAMESPACE
diff --git a/src/imports/location/qdeclarativegeoserviceprovider_p.h b/src/imports/location/qdeclarativegeoserviceprovider_p.h
index ecab260..0c377ae 100644
--- a/src/imports/location/qdeclarativegeoserviceprovider_p.h
+++ b/src/imports/location/qdeclarativegeoserviceprovider_p.h
@@ -43,7 +43,7 @@
 #define QDECLARATIVEQGEOSERVICEPROVIDER_H
 
 #include <qgeocoordinate.h>
-#include <QtDeclarative/qdeclarativeitem.h>
+#include <QtQuick1/qdeclarativeitem.h>
 
 #include <QMap>
 #include <QString>
diff --git a/src/imports/location/qdeclarativegraphicsgeomap_p.h b/src/imports/location/qdeclarativegraphicsgeomap_p.h
index 603d827..5b217ed 100644
--- a/src/imports/location/qdeclarativegraphicsgeomap_p.h
+++ b/src/imports/location/qdeclarativegraphicsgeomap_p.h
@@ -44,7 +44,7 @@
 
 #include <qgraphicsgeomap.h>
 #include <QPointer>
-#include <QtDeclarative/qdeclarativeitem.h>
+#include <QtQuick1/qdeclarativeitem.h>
 
 #include "qdeclarativegeomapobject_p.h"
 
