diff --git a/qtmultimedia.pro b/qtmultimedia.pro
index 00cd6bd..56fa9c4 100644
--- a/qtmultimedia.pro
+++ b/qtmultimedia.pro
@@ -6,17 +6,13 @@ module_qtmultimedia_src.target = module-qtmultimedia-src
 module_qtmultimedia_demos.subdir = demos
 module_qtmultimedia_demos.target = module-qtmultimedia-demos
 module_qtmultimedia_demos.depends = module_qtmultimedia_src
-!contains(QT_BUILD_PARTS,demos) {
-    module_multimedia_demos.CONFIG += no_default_install no_default_target
-}
+module_multimedia_demos.CONFIG += no_default_install no_default_target
 
 
 module_qtmultimedia_examples.subdir = examples
 module_qtmultimedia_examples.target = module-qtmultimedia-examples
 module_qtmultimedia_examples.depends = module_qtmultimedia_src
-!contains(QT_BUILD_PARTS,examples) {
-    module_multimedia_examples.CONFIG += no_default_install no_default_target
-}
+module_multimedia_examples.CONFIG += no_default_install no_default_target
 
 
 module_qtmultimedia_tests.subdir = tests
