From 084079a8133ea7afec121ae7b8aee2700bb613fd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mika=20Bostr=C3=B6m?= <bostik@odin.bostik.iki.fi>
Date: Fri, 26 Oct 2012 20:18:53 +0300
Subject: [PATCH 5/9] Disable tools entirely

---
 WebKit.pro |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/WebKit.pro b/WebKit.pro
index 74fa37c..e81dcfe 100644
--- a/WebKit.pro
+++ b/WebKit.pro
@@ -49,6 +49,6 @@ SUBDIRS += QtWebKit
 
 Tools.file = Tools/Tools.pro
 Tools.makefile = Makefile.Tools
-SUBDIRS += Tools
+#SUBDIRS += Tools
 
 # Number of times incremental builds have failed: 0
-- 
1.7.10.4

