From 52a7e0f45d415ec885cdeba89c68201140c58b99 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mika=20Bostr=C3=B6m?= <bostik@odin.bostik.iki.fi>
Date: Fri, 26 Oct 2012 20:18:53 +0300
Subject: [PATCH 10/10] Disable tools entirely

---
 WebKit.pro |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/WebKit.pro b/WebKit.pro
index 320e8a2..df53447 100644
--- a/WebKit.pro
+++ b/WebKit.pro
@@ -72,4 +72,5 @@ QtWebKit.file = Source/QtWebKit.pro
 QtWebKit.makefile = Makefile.QtWebKit
 SUBDIRS += QtWebKit
 
-SUBDIRS += Tools
+# Nope, not this one
+#SUBDIRS += Tools
-- 
1.7.10.4

