diff -Naur qt3d/src/imports/shapes/shapes.pro qt3db/src/imports/shapes/shapes.pro
--- qt3d/src/imports/shapes/shapes.pro	2012-02-15 13:46:42.142404015 +0200
+++ qt3db/src/imports/shapes/shapes.pro	2012-02-26 15:54:22.414814635 +0200
@@ -2,9 +2,11 @@
 TARGET  = qshapesqmlplugin
 CONFIG += qt plugin
 
-QT += declarative quick 3d 3dquick
+QT += declarative quick qt3d quick3d
 
-DESTDIR = $$QT.3dquick.imports/Qt3D/Shapes
+include(../../threed/includes.pri)
+
+#DESTDIR = $$QT.3dquick.imports/Qt3D/Shapes
 target.path = $$[QT_INSTALL_IMPORTS]/Qt3D/Shapes
 INSTALLS += target
 
diff -Naur qt3d/src/imports/threed/threed.pro qt3db/src/imports/threed/threed.pro
--- qt3d/src/imports/threed/threed.pro	2012-02-15 13:46:42.142404015 +0200
+++ qt3db/src/imports/threed/threed.pro	2012-02-26 15:43:52.603691563 +0200
@@ -4,7 +4,9 @@
 
 QT += quick declarative 3d 3dquick
 
-DESTDIR = $$QT.3dquick.imports/Qt3D
+include(../../threed/includes.pri)
+
+#DESTDIR = $$QT.3dquick.imports/Qt3D
 target.path = $$[QT_INSTALL_IMPORTS]/Qt3D
 INSTALLS += target
 
diff -Naur qt3d/src/plugins/sceneformats/assimp/assimp.pro qt3db/src/plugins/sceneformats/assimp/assimp.pro
--- qt3d/src/plugins/sceneformats/assimp/assimp.pro	2012-02-15 13:46:42.146404039 +0200
+++ qt3db/src/plugins/sceneformats/assimp/assimp.pro	2012-02-26 16:03:35.533557401 +0200
@@ -1,5 +1,6 @@
 TARGET = qsceneai
 include(../../qpluginbase.pri)
+include(../../../threed/includes.pri)
 HEADERS += qailoader.h \
     qaiscene.h \
     qaiscenehandler.h \
@@ -16,7 +17,7 @@
 
 QT += 3d
 
-DESTDIR = $$QT.3d.plugins/sceneformats
+#DESTDIR = $$QT.3d.plugins/sceneformats
 target.path = $$[QT_INSTALL_PLUGINS]/sceneformats
 INSTALLS += target
 
diff -Naur qt3d/src/plugins/sceneformats/bezier/bezier.pro qt3db/src/plugins/sceneformats/bezier/bezier.pro
--- qt3d/src/plugins/sceneformats/bezier/bezier.pro	2012-02-15 13:46:42.146404039 +0200
+++ qt3db/src/plugins/sceneformats/bezier/bezier.pro	2012-02-26 16:03:08.677424223 +0200
@@ -1,5 +1,6 @@
 TARGET  = qscenebezier
 include(../../qpluginbase.pri)
+include(../../../threed/includes.pri)
 
 HEADERS += qglbezierscene.h \
            qglbezierscenehandler.h
@@ -9,6 +10,6 @@
 
 QT += 3d
 
-DESTDIR = $$QT.3d.plugins/sceneformats
+#DESTDIR = $$QT.3d.plugins/sceneformats
 target.path = $$[QT_INSTALL_PLUGINS]/sceneformats
 INSTALLS += target
diff -Naur qt3d/src/quick3d/quick3d.pro qt3db/src/quick3d/quick3d.pro
--- qt3d/src/quick3d/quick3d.pro	2012-02-15 13:46:42.150404055 +0200
+++ qt3db/src/quick3d/quick3d.pro	2012-02-26 15:33:00.704458968 +0200
@@ -18,6 +18,8 @@
 }
 
 include(quick3d.pri)
+include(../threed/includes.pri)
+LIBS += -L../threed/ -lQt3D
 
 PUBLIC_HEADERS = $$HEADERS
 HEADERS += $$PRIVATE_HEADERS
diff -Naur qt3d/src/threed/includes.pri qt3db/src/threed/includes.pri
--- qt3d/src/threed/includes.pri	1970-01-01 02:00:00.000000000 +0200
+++ qt3db/src/threed/includes.pri	2012-02-26 14:34:40.447102139 +0200
@@ -0,0 +1,15 @@
+# Add everything in threed/* to include paths
+INCLUDEPATH += $$PWD/../quick3d
+INCLUDEPATH += $$PWD/scene
+INCLUDEPATH += $$PWD/effects
+INCLUDEPATH += $$PWD/geometry
+INCLUDEPATH += $$PWD/arrays
+INCLUDEPATH += $$PWD/global
+INCLUDEPATH += $$PWD/math3d
+INCLUDEPATH += $$PWD/materials
+INCLUDEPATH += $$PWD/textures
+INCLUDEPATH += $$PWD/painting
+INCLUDEPATH += $$PWD/surfaces
+INCLUDEPATH += $$PWD/viewing
+INCLUDEPATH += $$PWD/graphicsview
+
diff -Naur qt3d/src/threed/threed.pro qt3db/src/threed/threed.pro
--- qt3d/src/threed/threed.pro	2012-02-15 13:46:42.170404154 +0200
+++ qt3db/src/threed/threed.pro	2012-02-26 14:21:50.399283684 +0200
@@ -19,6 +19,7 @@
 
 include(../private/private.pri)
 include(threed.pri)
+include(includes.pri)
 PUBLIC_HEADERS = $$HEADERS
 HEADERS += $$PRIVATE_HEADERS
 DEFINES += QT_BUILD_QT3D_LIB
