diff -Naur qt3d/src/quick3d/quick3d.pro qt3d-/src/quick3d/quick3d.pro
--- qt3d/src/quick3d/quick3d.pro	2012-03-06 08:23:00.020430461 +0200
+++ qt3d-/src/quick3d/quick3d.pro	2012-03-06 10:13:20.337258841 +0200
@@ -25,8 +25,10 @@
 
 !contains(QT_CONFIG, egl):DEFINES += QT_NO_EGL
 
+devel.files = $$PUBLIC_HEADERS
+devel.path = $$[QT_INSTALL_HEADERS]/$$TARGET
 
-
+INSTALLS += devel
 
 
 
diff -Naur qt3d/src/threed/threed.pro qt3d-/src/threed/threed.pro
--- qt3d/src/threed/threed.pro	2012-02-15 13:46:42.170404154 +0200
+++ qt3d-/src/threed/threed.pro	2012-03-06 10:13:47.101391561 +0200
@@ -24,3 +24,9 @@
 DEFINES += QT_BUILD_QT3D_LIB
 
 !contains(QT_CONFIG, egl):DEFINES += QT_NO_EGL
+
+devel.files = $$PUBLIC_HEADERS
+devel.path = $$[QT_INSTALL_HEADERS]/$$TARGET
+
+INSTALLS += devel
+
