diff --git a/qtmultimedia.pro b/qtmultimedia.pro
index 580a98a..8d754e8 100644
--- a/qtmultimedia.pro
+++ b/qtmultimedia.pro
@@ -17,6 +17,5 @@ module_qtmultimedia_tests.CONFIG = no_default_install
 !contains(QT_BUILD_PARTS,tests):module_qtmultimedia_tests.CONFIG += no_default_target
 
 SUBDIRS += module_qtmultimedia_src \
-           module_qtmultimedia_examples \
            module_qtmultimedia_tests \
 
