From b8b51bfa78697f855b697e0135df7ae17cac2385 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mika=20Bostr=C3=B6m?= <bostik@odin.bostik.iki.fi>
Date: Mon, 12 Dec 2011 11:20:52 +0200
Subject: [PATCH 11/13] Remove extra stuff from build

---
 qtquick3d.pro |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/qtquick3d.pro b/qtquick3d.pro
index cb9ad31..d75ee37 100644
--- a/qtquick3d.pro
+++ b/qtquick3d.pro
@@ -1,12 +1,11 @@
 TEMPLATE = subdirs
 SUBDIRS += src
-!gcov: SUBDIRS += util examples demos
+!gcov: SUBDIRS += util
 
 include(pkg.pri)
 
 !package: SUBDIRS += tutorials
 
-SUBDIRS += tests
 CONFIG += ordered
 
 include(doc/doc.pri)
-- 
1.7.7.3

