diff --git a/src/assistant/assistant/assistant.pro b/src/assistant/assistant/assistant.pro
index eeee0ad..47a25df 100644
--- a/src/assistant/assistant/assistant.pro
+++ b/src/assistant/assistant/assistant.pro
@@ -119,3 +119,7 @@ contains(CONFIG, static): {
         DEFINES += USE_STATIC_SQLITE_PLUGIN
     }
 }
+
+# libQtHelp has been built into lib/
+# libQtCLucene is needed when linking QtHelp
+LIBS += -L../../lib -L../../lib/fulltextsearch -lQtCLucene
