diff --git a/src/designer/src/components/lib/lib.pro b/src/designer/src/components/lib/lib.pro
index eb9d31c..35a40eb 100644
--- a/src/designer/src/components/lib/lib.pro
+++ b/src/designer/src/components/lib/lib.pro
@@ -7,7 +7,7 @@ CONFIG += qt depend_prl no_objective_c designer
 QT *= gui-private
 QT *= uilib-private
 win32|mac: CONFIG += debug_and_release
-DESTDIR = $$QT_BUILD_TREE/lib
+DESTDIR = ../../../../../lib
 !wince*:DLLDESTDIR = $$QT.designer.bins
 QT += designer-private uilib-private
 
