diff --git a/src/designer/src/lib/lib.pro b/src/designer/src/lib/lib.pro
index 3c386cf..e4cfc14 100644
--- a/src/designer/src/lib/lib.pro
+++ b/src/designer/src/lib/lib.pro
@@ -8,7 +8,7 @@ QT += core-private gui-private xml uilib-private uilib-private
 contains(QT_CONFIG, reduce_exports):CONFIG += hide_symbols
 CONFIG += qt
 win32|mac: CONFIG += debug_and_release
-DESTDIR = $$QT_BUILD_TREE/lib
+DESTDIR = ../../../../lib
 !wince*:DLLDESTDIR = $$QT.designer.bins
 
 INCLUDEPATH += $$QT.designer.includes \
